apps/dashboard/src/app.postcss

```diff
@@ -203,6 +203,16 @@ code {
   border-color: #fff1f2 !important;
 }
 
+.theme-custom .bg-primary-50 {
+  background-color: #fff1f2 !important;
+}
+.theme-custom .text-primary-50 {
+  color: #fff1f2 !important;
+}
+.theme-custom .border-primary-50 {
+  border-color: #fff1f2 !important;
+}
```

Comment:
@tunny17 This should probably not be here

Reply:
yes, my bad, i'll remove that in this fix i'm about to push